app/controllers/index.js

```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
     return this.filteredEvents ? this.filteredEvents.filter(event => {return event.isFeatured}) : null;
 
   }),
+    marker: {
+    type: 'FeatureCollection',
+    features: [
+      {
+        type: 'Feature',
+        geometry: { type: 'Point', coordinates: [ -96.7969879, 32.7766642 ] }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)